src/main/java/de/retest/recheck/ui/descriptors/SutState.java

```diff
@@ -22,25 +25,42 @@
 	@XmlElement
 	private final List<RootElement> descriptors;
 
+	@XmlElement
+	private final Map<String, String> metadata;
```

Comment:
I think that my comment got lost:

Since this changes the model, do we need to increase the `persistenceVersion` just in case? I think this should be compatible with old Golden Masters, but just to be sure that we could include the corresponding transformers for that in case.

We could do that separate if we realized that there are in fact incompatibilities.

Reply:
This should be a backwards compatible change (added field). Alsot, otherwise tests should have failed.